src/rstar.jl

```diff
@@ -4,8 +4,8 @@
 
 Compute the R* convergence diagnostic of MCMC.
 
-This implementation is an adaption of Algorithm 1 & 2, described in [^LambertVehtari]. Note
-that the correctness of the statistic depends on the convergence of the classifier used
+This implementation is an adaption of Algorithm 1 & 2, described by Lambert & Vehtari ([2020](https://arxiv.org/abs/2003.07900)).
```

Comment:
Why was this changed? The previous version used the Documenter format for references.

Reply:
The references were not consistent; Most used a url and some text and this was the only one which used a footnote. The footnote looks looks like this in the text:

![image](https://user-images.githubusercontent.com/20724914/108840102-2cccc300-75d6-11eb-9bc5-035e189936f8.png)

and like this in at the bottom of the page:

![image](https://user-images.githubusercontent.com/20724914/108840684-05c2c100-75d7-11eb-9245-28bf638486cd.png)

A relevant discussion is also at [discourse](https://discourse.julialang.org/t/citations-in-docs/5066/37). Also, Julia shows them just in text at https://docs.julialang.org/en/v1/base/math/#Base.Math.hypot. Anyway, I don't have a strong opinion on this. Let me know what you prefer and I'll change it to that style.

Reply:
On the same page, https://docs.julialang.org/en/v1/base/math/#Base.Math.evalpoly shows an example where base Julia uses the Documenter-style references :slightly_smiling_face: